test_project/tests/ui/test_spotify_tracks.py

```diff
@@ -11,16 +13,14 @@
         ('The Beatles', 'Here Comes The Sun - Remastered 2009')
     ]
 )
-def test_spotify_tracks(browser, artist, song_name):
-    home_page = HomePage(browser)
-    home_page.open('https://spotify.com')
-    assert home_page.is_cookies_button_displayed(), 'Accept cookies button is not displayed'
-    home_page.click_cookies_button()
-    home_page.wait_until_cookies_button_is_not_displayed()
-    assert not home_page.is_cookies_button_displayed(), 'Accept cookies button is displayed'
+def test_spotify_tracks(page: Page, artist, song_name):
```

Comment:
could we make Page a singleton stored in some container and initialized through fixture? Could we avoid passing it to every constructor?

Reply:
fixed